src/Workspaces/Core/Portable/Notification/GlobalOperationEventArgs.cs

```diff
@@ -1,17 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Immutable;
-
-namespace Microsoft.CodeAnalysis.Notification
-{
-    internal class GlobalOperationEventArgs : EventArgs
-    {
-        public ImmutableArray<string> Operations { get; }
```

Comment:
these values were never read.  so i removed htis EventArgs subclass and replaced with EventArgs.